private/pkg/manifest/module.go

```diff
@@ -166,6 +187,15 @@ func (s *BlobSet) BlobFor(digest string) (Blob, bool) {
 	return blob, true
 }
 
+// Blobs returns the blobs in the set as an array.
+func (s *BlobSet) Blobs() []Blob {
+	var blobs []Blob
```

Comment:
```suggestion
	blobs := make([]Blob, len(s.digestToBlob))
```
... should reduce memory allocations.